site/ipam/troubleshooting-ipam.md

```diff
@@ -6,20 +6,18 @@ layout: default
 
 The command
 
+~~~bash
     weave status
+~~~
 
 reports on the current status of the weave router and IP allocator:
 
-````
-...
-
+~~~bash
        Service: ipam
         Status: awaiting consensus (quorum: 2, known: 0)
          Range: 10.32.0.0-10.47.255.255
  DefaultSubnet: 10.32.0.0/12
-
-...
-````
+~~~
```

Comment:
There were some leading and trailing `...` here, to indicate that there is more information displayed before and after.